p2p/pex/pex_reactor_test.go

```diff
@@ -259,44 +226,66 @@ func TestPEXReactorAddrsMessageAbuse(t *testing.T) {
 }
 
 func TestPEXReactorUsesSeedsIfNeeded(t *testing.T) {
+	require := require.New(t)
```

Comment:
I think we want to avoid this. I don't really mind either way, but for consistency with the rest of the code. Also, it's not great to have variables with the same name as packages and it's not really any trouble to include `t` as an argument to things

Reply:
Thought this is the preferred way, looking at most other test cases (especially the ones in the same file). I agree that package names shouldn't be shadowed by vars in the local scop, but also would prefer a consistent style, which currently seems to be this one.

Reply:
Whoops you're right. We've had some back and forth on this. Let's work towards passing `t` explicitly in the calls.

Reply:
On board with that. :ship: